gradle.properties

```diff
@@ -1,5 +1,5 @@
-version_major=0
-version_minor=0
+version_major=8
+version_minor=3
```

Comment:
Please revert

Reply:
Ah fuck, accident. I had to do this locally to make it load in ATM for testing :-(